lib/spotify_handler.coffee

```diff
@@ -267,6 +267,17 @@ class SpotifyHandler
     @storage.setItem 'last_playlist', name
     return
 
+  list_random: ->
+    if Math.random() > 0.5
```

Comment:
Oops don't know how that slipped in, I updated the code to remove this unintented behaviour